src/constants/constants.cpp

```diff
@@ -0,0 +1,94 @@
+/***
+ * Maracatronics Robotics
+ * Federal University of Pernambuco (UFPE) at Recife
+ * http://www.maracatronics.com/
+ *
+ * This file is part of Armorial project.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ ***/
+
+#include "constants.h"
+
+Constants::Constants(QString fileName) {
+    _fileName = fileName;
+
+    // Opening and reading file content in buffer
+    _file.setFileName(_fileName);
+    _file.open(QIODevice::ReadOnly | QIODevice::Text);
+    _fileBuffer = _file.readAll();
+    _file.close();
+
+    // Parsing buffer to json objects
+    _document = QJsonDocument::fromJson(_fileBuffer.toUtf8());
+    _documentMap = _document.object().toVariantMap();
+
+    // Read data
+    readThreadConstants();
```

Comment:
Always follow this model to load constants to an module:
- Create an **readXConstants()** module
- Create an **QVariantMap** following those examples
- Load every data from it an output this data (for debug purposes)